machine-learning/src/main.py

```diff
@@ -38,6 +42,17 @@ def image_classification():
     assetPath = request.json['thumbnailPath']
     return run_engine(classifier, assetPath), 201
 
+@server.route("/sentence-transformer/encode-image", methods=['POST'])
+def clip_encode_image():
+    assetPath = request.json['thumbnailPath']
+    emb = sentence_transformer_model.encode(Image.open(assetPath))
+    return emb.tolist(), 201
+
+@server.route("/sentence-transformer/encode-text", methods=['POST'])
+def clip_encode_text():
+    text = request.json['text']
+    emb = sentence_transformer_model.encode(text)
+    return emb.tolist(), 201
```

Comment:
Since we're now downloading and caching models on container start, and so image size is no longer a concern, do we want to default to using the multilingual CLIP model for text embeddings?

Reply:
I would say probably make it a configurable option to enable it

Reply:
I've made all the models env variables now, so users should be able to override the default one.

Reply:
Looking at the example, maybe it makes sense to do 2 env variables for the CLIP? One for the text and another one for the images. At least that's how it's handled in the example in [sentence-transformers](https://github.com/UKPLab/sentence-transformers/blob/master/examples/applications/image-search/Image_Search-multilingual.ipynb)

Reply:
Nice, I've made them separate now.

Reply:
This is awesome! Though I would guess that the majority of users will leave the default env variable. Maybe it's worth adding a check that'll avoid loading second model, in case we have same models for image and text encoding?

Reply:
I haven't tested this, but I think I understand what you are saying - something like this?
```python
clip_image_model = SentenceTransformer(image_model)
clip_text_model = clip_image_model
if image_model != text_model:
  clip_text_model = SentenceTransformer(text_model)
```

Reply:
Yes, I'm pretty sure something like this will do the job. Maybe inline `if` would be a bit more neat, i.e. like
```python
clip_image_model = SentenceTransformer(image_model)
clip_text_model = clip_image_model if image_model == text_model else SentenceTransformer(text_model)
```

Reply:
I'm not a python expert, but that's actually exactly how I implemented it (after googling to see if there was a ternanry way lol).

Reply:
They're lazy loaded an deduped now.